util/a2e/atoe_utils.c

```diff
@@ -115,12 +115,12 @@ pchar(InstanceData *this, int c)
  *               str            String to add to buffer
  *               left_justify   Left justify string flag
  *               min_width      Minimum width of string added to buffer
- *               precision
+ *               max_width      Maximum chars to take from the argument
```

Comment:
I think we should keep the name `precision`. See [1] which uses that term.

[1] https://man7.org/linux/man-pages/man3/fprintf.3p.html

Reply:
good point! I reverted the name change